tests/extras/test_ftrl.py

```diff
@@ -43,9 +43,9 @@
 tparams = Params(alpha = 1, beta = 2, lambda1 = 3, lambda2 = 4, d = 5,
                      n_epochs = 6, inter = True)
 
-tmodel = dt.Frame([[random.random() for i in range(tparams.d)],
-                      [random.random() for i in range(tparams.d)]],
-                      names=['z', 'n'])
+tmodel = dt.Frame([[random.random()] * tparams.d,
+                   [random.random()] * tparams.d],
```

Comment:
Here, the old statement was more correct:
`[random.random() for _ in range(N)]` creates a list of N different random numbers;
`[random.random()] * N` creates a list of N same numbers, although that number is chosen randomly.

Reply:
Both will probably work in this case, however, may be for the purpose of testing different random numbers are better. Will revert it back, thanks.